README.md

```diff
@@ -212,7 +212,7 @@ The test tool has the following limitations. If the DUT falls under these limita
 
 | **Requirement** | **T2IAPI Manipulation** |
 |-----------------|-------------------------|
-| R0036           | SetLocationDetail       |
+| R0036_0         | SetLocationDetail       |
```

Comment:
At the moment its still R0036.

Reply:
Ok, I'll move this change to #33

Reply:
Turns out: It is already [part of #33](https://github.com/Draegerwerk/SDCcc/pull/33/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5).